numcodecs/blosc.pyx

```diff
@@ -483,9 +577,15 @@ class Blosc(Codec):
         return compress(buf, self._cname_bytes, self.clevel, self.shuffle, self.blocksize)
 
     def decode(self, buf, out=None):
+        '''**Experimental**'''
```

Comment:
I have it in both. I put this one here as well since I figure a user would be more likely to use this method then the function. Should this be removed?

Reply:
I think there is a confusion, this docstring is on the `decode` method, I think it's meant to be on the `decode_partial` method.

Reply:
Oh whoops, now I see. Just fixed it!